noxfile.py

```diff
@@ -90,13 +98,15 @@ def coverage_release(session: nox.Session):
     """
     install(
         session,
-        ".",
+        ".[coverage]",
         *other_antsibull(),
         "ansible-core",
-        "coverage",
         editable=True,
     )
 
+    tmp = Path(session.create_tmp())
+    covfile = tmp / ".coverage"
+    env = {"COVERAGE_FILE": f"{covfile}", **session.env}
```

Comment:
I'd rename this varable to `cov_env` or `coverage_env` or something to make its purpose clearer, but that's up to you.

Reply:
f99a6205f213a4524fdbe24cc3b863b180b885fe.